concepticondata/conceptlists.tsv

```diff
@@ -211,4 +211,5 @@ Voorhoeve-1971-125	Voorhoeve, C.L.	1971		125	basic,areal	English	New Guinea		Voo
 Foley-1986-50	Foley, W. A.	1986		50	basic,areal	English	New Guinea		Foley1986	Foley1986	A 50 item wordlist chosen for illustrative purposes.	215-216	
-Luniewska-2016-299	Łuniewska, Magdalena AND Haman, Ewa AND Armon-Lotem, Sharon AND Etenkowski, Bartłomiej AND Southwood, Frenette AND Anđelković, Darinka AND Blom, Elma AND Boerma, Tessel AND Chiat, Shula AND Engel de Abreu, Pascale AND Gagarina, Natalia AND Gavarró, Anna AND Håkansson, Gisela AND Hickey, Tina AND Jensen de López, Kristine AND Marinis, Theodoros AND Popović, Maša AND Thordardottir, Elin AND Blažienė, Agnė AND Cantú Sánchez, Myriam AND Dabašinskienė, Ineta AND Ege, Pınar AND Ehret, Inger-Anne AND Fritsche, Nelly-Ann AND Gatt, Daniela AND Janssen, Bibi AND Kambanaros, Maria AND Kapalková, Svetlana AND Kronqvist, Bjarke AND Kunnari, Sari AND Levorato, Chiara AND Nenonen, Olga AND Nic Fhlannchadha, Siobhán AND O’Toole, Ciara AND Polišenská, Kamila AND Pomiechowska, Barbara AND Ringblom, Natalia AND Rinker, Tanja AND Roch, Maja AND Savić, Maja AND Slančová, Daniela AND Tsimpli, Ianthi Maria AND Ünal-Logacev, Özlem	2016		299	acquisition	Afrikaans, Catalan, Danish, Dutch, English, Finnish, German, Greek, Hebrew, Hungarian, Icelandic, Irish, Xhosa, Italian, Lithuanian, Luxembourgish, Maltese, Polish, Russian, SouthAfricanEnglish, Serbian, Slovak, Spanish, Swedish, Turkish	Global	https://link.springer.com/article/10.3758%2Fs13428-015-0636-6	Luniewska2016			1154-1177	
\ No newline at end of file
+Luniewska-2016-299	Łuniewska, Magdalena AND Haman, Ewa AND Armon-Lotem, Sharon AND Etenkowski, Bartłomiej AND Southwood, Frenette AND Anđelković, Darinka AND Blom, Elma AND Boerma, Tessel AND Chiat, Shula AND Engel de Abreu, Pascale AND Gagarina, Natalia AND Gavarró, Anna AND Håkansson, Gisela AND Hickey, Tina AND Jensen de López, Kristine AND Marinis, Theodoros AND Popović, Maša AND Thordardottir, Elin AND Blažienė, Agnė AND Cantú Sánchez, Myriam AND Dabašinskienė, Ineta AND Ege, Pınar AND Ehret, Inger-Anne AND Fritsche, Nelly-Ann AND Gatt, Daniela AND Janssen, Bibi AND Kambanaros, Maria AND Kapalková, Svetlana AND Kronqvist, Bjarke AND Kunnari, Sari AND Levorato, Chiara AND Nenonen, Olga AND Nic Fhlannchadha, Siobhán AND O’Toole, Ciara AND Polišenská, Kamila AND Pomiechowska, Barbara AND Ringblom, Natalia AND Rinker, Tanja AND Roch, Maja AND Savić, Maja AND Slančová, Daniela AND Tsimpli, Ianthi Maria AND Ünal-Logacev, Özlem	2016		299	acquisition	Afrikaans, Catalan, Danish, Dutch, English, Finnish, German, Greek, Hebrew, Hungarian, Icelandic, Irish, Xhosa, Italian, Lithuanian, Luxembourgish, Maltese, Polish, Russian, SouthAfricanEnglish, Serbian, Slovak, Spanish, Swedish, Turkish	Global	https://link.springer.com/article/10.3758%2Fs13428-015-0636-6	Luniewska2016			1154-1177	
+Gampe-2017-48	Gampe, Anja AND Kurthen, Ira AND Daum, Moritz M.	2017		48	acquisition	English	Global	http://journals.sagepub.com/doi/full/10.1177/0142723717736450	Gampe2017				
```

Comment:
Wait, sorry for not mentioning earlier, but we'll need a little story here, explaining, for example, that Gampe's list is partially building on Luniewska, if I'm not mistaken.

Reply:
Yes, one more thing: this should also include some explanation of the new columns in the data. I am not sure, would have to re-read the paper, but I think it'll be some reaction times for the mean_concept_vocab? Maybe, changing the name of the column to be more telling, might also not be a bad idea.

Reply:
Alright, will write notes & rename columns.